### PR TITLE
listcase rewrite

### PR DESCRIPTION
**Pull request by Nyameliaaaa**

### CODE REVIEW

**Comment by Nyameliaaaa** [comment]
- [ ] fix the goddamn array

**Comment by Nyameliaaaa** [comment]
UNSEX

- [ ] fix it locking up

**Comment by Nyameliaaaa** [comment]
UNSEX 2

early aeon pag moment

File: src/commands/listcase.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            .setLabel(client.language.get(interaction.locale, 'LISTCASE_NEXT'))
+            .setStyle('PRIMARY')
+    );
+
+    interaction.reply({ embeds: [embed], components: [row] });
+}
+
+export const data = new SlashCommandBuilder()
+    .setName('listcase')
+    .setDescription('Check punishments of a user')
```
Comment (SuperchupuDev) [inline src/commands/listcase.ts:99]:
```suggestion
    .setDescription('Check punishments of an user')
```

Comment (Nyameliaaaa) [reply src/commands/listcase.ts:99]: `a user` is the correct one. see [this stackexchange](https://english.stackexchange.com/questions/4700/is-it-a-uniform-or-an-uniform) for info

**Review by hkubus — DISMISSED** [review]

**Review by hkubus — DISMISSED** [review]
agree

**Review by SuperchupuDev — APPROVED** [review]

**Review by hkubus — APPROVED** [review]

**Pull request merged**